ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh

```diff
@@ -61,3 +61,19 @@ oc apply -f /tmp/kube_objects.yaml -n "$OO_INSTALL_NAMESPACE"
 
 # Remove the kube objects file just in case
 rm -rf /tmp/kube_objects.yaml
+
+
+KEYWORD_CUSTOM_KUBEOBJECTS="custom-"
+
+CUSTOM_KUBEOBJECTS_PATH=${CUSTOM_KUBEOBJECTS_PATH:=/var/run/}
+
+# the following command lists all the kubeobjects mounted on CUSTOM_KUBEOBJECTS_PATH 
+# with prefix "custom-" and also package name within the name of the file.
+
+LIST_OF_KUBEOBJECTS=$(find "${CUSTOM_KUBEOBJECTS_PATH}${KEYWORD_CUSTOM_KUBEOBJECTS}${OO_PACKAGE}-"* -type f -name kube_objects)
```

Comment:
Please remove the last `-` as this should be optional

Reply:
@apodhrad Which - are we talking about ? 
is it in the name ?

Reply:
`"${CUSTOM_KUBEOBJECTS_PATH}${KEYWORD_CUSTOM_KUBEOBJECTS}${OO_PACKAGE}"*`

instead of 

`"${CUSTOM_KUBEOBJECTS_PATH}${KEYWORD_CUSTOM_KUBEOBJECTS}${OO_PACKAGE}-"*`

Reply:
Sure